grpc.go

```diff
@@ -23,6 +23,9 @@ import (
 	"github.com/sirupsen/logrus"
 	"golang.org/x/net/context"
 	"golang.org/x/sys/unix"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
```

Comment:
Please rename the import with something like:
```go
import (
        ...
        grpcStatus "google.golang.org/grpc/status"
)
```
This way, we won't have to rename the variables `status` which I like.
You could do the same with `codes` into `grpcCodes` so that it won't conflict in the future.